qcommon/src/main/java/jake2/qcommon/network/messages/server/ConfigStringMessage.java

```diff
@@ -35,7 +37,47 @@ public int getSize() {
 
     @Override
     public String toString() {
-        return "ConfigStringMessage{" + index + "=" + config + '}';
+        return "ConfigStringMessage{" + getType(index) + "(" + index + ")" + "=" + config + '}';
+    }
+
+    public String getType(int index) {
+        if (index == CS_NAME) {
```

Comment:
I wonder if there is a way to generate this switch

Reply:
not in it's current form